src/pages/home/report/ReportActionItem.js

```diff
@@ -741,6 +764,11 @@ function ReportActionItem(props) {
         return null;
     }
 
+    // if action is actionable join by invited link and resolved by user, then we don't want to render anything
+    if (ReportActionsUtils.isActionableJoinRequest(props.action) && lodashGet(props.action, 'originalMessage.choice', '')) {
```

Comment:
We still want to render `<User> requested to join <Workspace>`, we just want to hide the buttons